api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export async function loginUser(email, password) {
+  try {
+    const response = await axios.post('/api/login', { email, password });
+    return response.data;
+  } catch (error) {
```

Comment:
Consider throwing the error after catching it so that the caller of this function can handle it appropriately.